fedimint-server/Cargo.toml

```diff
@@ -15,6 +15,7 @@ path = "src/lib.rs"
 [dependencies]
 fedimint-aead = { path = "../crypto/aead" }
 anyhow = "1.0.66"
+async-channel = "1.8.0"
```

Comment:
https://docs.rs/tokio/latest/tokio/sync/mpsc/index.html instead? Or you need mpmc? Or perf?

Reply:
We need mpmc